### DISCUSSION TITLE
Build for web but got error

### DISCUSSION THREAD

**Discussion by NoverNobida** [rustdesk/rustdesk#3286] [Q&A]
Build for web with doc https://rustdesk.com/docs/en/dev/build/web/, but got error:
```
Traceback (most recent call last):
  File "./gen_js_from_hbb.py", line 77, in <module>
    main()
  File "./gen_js_from_hbb.py", line 29, in main
    assert(len(toks) == 2)
AssertionError
```
Is there any dependency missing?

**Comment by tuvok81** [comment]
i have the same Problem .. gives anybody where can help here? 

![grafik](https://github.com/rustdesk/rustdesk/assets/30367063/1df42fa8-6150-4216-82c7-06cc85fbf8cb)

**Comment by Anthael** [comment]
The problem seems to be linked to your python version.
Have you tried to execute the command 
```bash
apt install python-is-python3
```
